docker/backend/Dockerfile

```diff
@@ -47,7 +47,7 @@ RUN ssh-keygen -f /etc/ssh/ssh_host_rsa_key -N '' -q
 
 # setup root user
 RUN echo 'root:passwd' | chpasswd && \
-    mkdir /root/.ssh && chmod 700 /root /root/.ssh
+    mkdir -p /root/.ssh && chmod 700 /root /root/.ssh
```

Comment:
This change shouldn't be needed.

Reply:
I needed it to make docker-compose work. It seemed to me that f39 has /root/.ssh dir by default. I'll delete images and I'll try it again if it is really needed

Reply:
I got 

```
 => ERROR [ 5/14] RUN echo 'root:passwd' | chpasswd &&     mkdir /root/.ssh && chmod 700 /root /root/.ssh                                                                                                       0.2s 
------                                                                                                                                                                                                               
 > [ 5/14] RUN echo 'root:passwd' | chpasswd &&     mkdir /root/.ssh && chmod 700 /root /root/.ssh:                                                                                                                  
0.132 mkdir: cannot create directory '/root/.ssh': File exists                                                                                                                                                       
------
Dockerfile:49
--------------------
  48 |     # setup root user
  49 | >>> RUN echo 'root:passwd' | chpasswd && \
  50 | >>>     mkdir /root/.ssh && chmod 700 /root /root/.ssh
  51 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c echo 'root:passwd' | chpasswd &&     mkdir /root/.ssh && chmod 700 /root /root/.ssh" did not complete successfully: exit code: 1
ERROR: Service 'backend-log' failed to build : Build failed
```

Reply:
I see ... I'd probably drop the `mkdir` then, if it is not needed?

This really seems like you use the `-p` becuase `/root` doesn't exist.

Reply:
IOW, `ssh-keygen` indeed creates `~/.ssh` (since ever?).  It is surprising this ever worked :-)

Reply:
I tested `ssh-keygen -f /etc/ssh/ssh_host_rsa_key -N '' -q` in a fresh f39 container, and `/root/.ssh` was not created.  openssh-clients-9.3p1-10.fc39.x86_64 weird.

Reply:
Ok I have no idea why it was created but I removed the mkdir since it works without it. Do you want to leave as it is without the mkdir or investigate what created it?

Reply:
I think it is created by installing openssh-server.  I think it is fine as-is (we should long-term avoid doing these permissions tweaks, I think, but it's for a separate fix).